main.go

```diff
@@ -296,8 +296,12 @@ func fetchIncidents(team string, since, until time.Time) ([]*incident, error) {
 			customerImpactScope:    *i.Attributes.CustomerImpactScope.Get(),
 			customerImpactDuration: time.Duration(*i.Attributes.CustomerImpactDuration * int64(time.Second)),
 			createdAt:              *i.Attributes.Created,
-			resolvedAt:             *i.Attributes.Resolved.Get(),
-		})
+		}
+		if i.Attributes.Resolved.IsSet() && i.Attributes.Resolved.Get() != nil {
```

Comment:
In my case `IsSet()` was true, but the timestamp was `nil` when performing the `Get()`.